0004-rfcs-over-adr-pdr/0004-rfcs-over-adr-pdr.md

```diff
@@ -0,0 +1,9 @@
+# RFC 0004 - RFCs over ADRs and PDRs
+
+This RFC describes the relationship between RFCs, ADRs, and PDRs.
+
+'ADR' refers to Architecture Decision Record, and 'PDR' refers to Product Decision Record. These records, as well as their processes, are currently located in the private Giant Swarm Intranet.
+
+ADRs and PDRs should be superseded by RFCs. This allows for one public process for discussing and recording ideas.
```

Comment:
```suggestion
ADRs and PDRs should be replaced by RFCs. This allows for one public process for discussing and recording ideas.
```

I had to look up what "superseded" means... 🙈

Reply:
Hm. But PDRs and ADRs merely document that a certain decision was made and why. Doesn't the RFC complement the ADR and PDR, but not replace them? i.e.

1. Open an RFC
2. People comment
3. We make a decision
4. Write PDR or ADR

Reply:
that is my understanding as well

Reply:
I like having the decisions public, but I would not like to change the format and mush them all together. The structure of the DRs serves its purpose and the non-structure of RFCs serves another purpose. RFCs might not even necessarily develop into DRs IMHO as we are not documenting every decision there is in architecture and product.